_docs/tasks/ingress.md

```diff
@@ -60,34 +60,51 @@ to configure ingress behavior.
    ```
    
    ```
-   NAME      HOSTS     ADDRESS                 PORTS     AGE
-   gateway   *         130.211.10.121          80        1d
-   export INGRESS_URL=130.211.10.121:80
+   NAME            HOSTS     ADDRESS                 PORTS     AGE
+   istio-ingress   *         130.211.10.121          80        1d
    ```
 
-   If load balancers are not supported, use the service NodePort instead:
+   ```bash
+   export INGRESS_URL=130.211.10.121
+   ```
+
+   If load balancers are not supported, use the ingress controller's hostIP:
+   
+   ```bash
+   kubectl get po -l istio=ingress -o jsonpath='{.items[0].status.hostIP}'
+   ```
+
+   ```
+   169.47.243.100
+   ```
+
+   along with the istio-ingress service's nodePort for port 80:
    
    ```bash
-   export INGRESS_URL=$(kubectl get po -l istio=ingress -o jsonpath='{.items[0].status.hostIP}'):$(kubectl get svc istio-ingress -o jsonpath='{.spec.ports[0].nodePort}')
+   kubectl get svc istio-ingress
+   ```
+   
+   ```
+   NAME            CLUSTER-IP     EXTERNAL-IP   PORT(S)                      AGE
+   istio-ingress   10.10.10.155   <pending>     80:31486/TCP,443:32254/TCP   32m
+
    ```
+   
+   ```bash
+   export INGRESS_URL=169.47.243.100:31486
 
+   ```
+   
 1. Access the httpbin service using _curl_:
 
    ```bash
    curl http://$INGRESS_URL/headers
    ```
 
-   ```json
+   ```
    {
      "headers": {
-       "Accept": "*/*", 
-       "Content-Length": "0", 
-       "Host": "httpbin.default.svc.cluster.local:8000", 
-       "User-Agent": "curl/7.35.0", 
-       "X-Envoy-Expected-Rq-Timeout-Ms": "15000", 
-       "X-Request-Id": "59cf4fce-72e0-4470-ade5-f59149705944"
-     }
-   }
+   ...
```

Comment:
The exact output is not relevant for this task, only that there is some output, so I thought it was best to remove the extra noise.